<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportLengthOfStayInHospitalDialog</class>
 <widget class="QDialog" name="ReportLengthOfStayInHospitalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="lblBegSetDateInfo">
     <property name="text">
      <string>Начало события</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegSetDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegSetDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLabel" name="lblEndSetDateInfo">
     <property name="text">
      <string>Начало события</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndSetDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип события</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblBegExecDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="edtBegExecDate"/>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLabel" name="lblEndExecDateInfo">
     <property name="text">
      <string>Окончание события</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblEndExecDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDateEdit" name="edtEndExecDate"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblMKB">
     <property name="text">
      <string>Коды диагнозов по &amp;МКБ</string>
     </property>
     <property name="buddy">
      <cstring>cmbMKBFilter</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLabel" name="lblBegExecDateInfo">
     <property name="text">
      <string>Окончание события</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cmbMKBFilter">
     <item>
      <property name="text">
       <string>Игнор.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Интервал</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="CICDCodeEdit" name="edtMKBFrom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string>a00.00; </string>
     </property>
     <property name="text">
      <string>A.</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="CICDCodeEdit" name="edtMKBTo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string>a00.00; </string>
     </property>
     <property name="text">
      <string>Z99.9</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegSetDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndSetDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndSetDate</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="cmbEventStatus">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>закрытые</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>открытые</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblEventStatus">
     <property name="text">
      <string>Статус события</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegSetDate</tabstop>
  <tabstop>edtEndSetDate</tabstop>
  <tabstop>cmbEventStatus</tabstop>
  <tabstop>edtBegExecDate</tabstop>
  <tabstop>edtEndExecDate</tabstop>
  <tabstop>cmbEventType</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbMKBFilter</tabstop>
  <tabstop>edtMKBFrom</tabstop>
  <tabstop>edtMKBTo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportLengthOfStayInHospitalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportLengthOfStayInHospitalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
